### DISCUSSION TITLE
[Office Hours]  Omit select user data in the Spezi framework based on privacy preferences enhancement

### DISCUSSION THREAD

**Discussion by dhruvna1k** [CS342/.github#53] [Office Hours]
### Related Discussion

https://github.com/CS342/2024-Prisma/issues/14

### Reproduction

Just wanted to follow up with Paul on how to make the code changes in the Spezi framework

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by PSchmiedmayer** [comment]
@dhruvna1k Thank you for coming by during the office hours today! It would be great if you can share some of the insights during the office hours here in a structured way to others that would have similar issues can look at the different resources we discussed 👍

**Reply by PSchmiedmayer** [reply]
@dhruvna1k Has your question already been resolved?
